### PR TITLE
feat: add events API requests

### PR DESCRIPTION
**Pull request by alexgarel**
see https://github.com/openfoodfacts/openfoodfacts-events

Replaces #674

### What

Call openfoodfacts-events each time a user annotate an item

### CODE REVIEW

**Comment by alexgarel** [comment]
Hi @ocervell I built upon your commit, but I changed it a bit.
- added a test
- only launch one process to have async posting (I think it was a bit too much to have one process for each events, also because we have in-process cache, which can make the fork costy after a while) 
- in dev setup: no post to events (I think few dev will have an events project setup)
- changed location of event sending to be able to have the barcode

I would be glad if you want to review it.

File: robotoff/app/api.py [hunk 1]
```diff
@@ -812,7 +812,8 @@ def on_post(self, req: falcon.Request, resp: falcon.Response):
         barcode = req.get_param("barcode", required=True)
         action = req.get_param("action", required=True)
         server_domain = req.get_param("server_domain", required=True)
-
+        print(server_domain)
+        print(settings.OFF_SERVER_DOMAIN)
```
Comment (ocervell) [inline robotoff/app/api.py:816]: remove the print statements before merging

**Review by ocervell — APPROVED** [review]
LGTM, but still needs testing within Docker local env and in dev env after !

**Comment by alexgarel** [comment]
Thanks for the review @ocervell 

I've done some test in local and it worked like a charm.

I will make a push to preprod to verify (through a specific PR on a deploy-* branch)

**Comment by alexgarel** [comment]
Tested ok in preprod (#718)

**Pull request merged**